tests/foreman/cli/test_import.py

```diff
@@ -29,6 +36,23 @@ def csv_to_dataset(csv):
     ]
 
 
+def build_csv_file(rows=[]):
```

Comment:
Do not use a mutable object as an argument to a function. You'll find lots of good explanations if you search for [python mutable argument](https://duckduckgo.com/?q=python+mutable+argument&t=ffsb&ia=qa). For example: http://stackoverflow.com/questions/1132941/least-astonishment-in-python-the-mutable-default-argument